_i18n/fr.yml

```diff
@@ -194,6 +194,27 @@ help:
 help_page:
   p_1: Parcourir les sujets communs
   title: Comment pouvons nous aider?
+  categories:
+    - title: Commencez avec login.gov
+      description: Créez votre compte. Découvrez les options d'authentification et d'autres caractéristiques du compte.
+      url: /help/get-started/overview
+      image: /assets/img/help/get-started.svg
+    - title: Des problèmes pour vous connecter?
+      description: Vous avez oublié votre mot de passe ? Votre compte est bloqué ? Nous vous aiderons à résoudre vos problèmes d'accès.
+      url: /help/trouble-signing-in/overview
+      image: /assets/img/help/trouble-signing-in.svg
+    - title: Gérez votre compte
+      description: Modifiez les paramètres de votre compte, notamment votre mot de passe, votre numéro de téléphone, votre adresse électronique, etc.
+      url: /help/manage-your-account/overview
+      image: /assets/img/help/manage-your-account.svg
+    - title: Assistance auprès des agences spécifiques
+      description: Obtenez de l'aide avec Trusted Traveler Programs (TTP), SAM.gov.
```

Comment:
Similar to a comment I made about Spanish but for French, we should add "et" — "... (TTP) et SAM.gov."

Reply:
I thought about this...but the translator omitted the "et". Wasn't sure if there was a reason for this.